.github/workflows/testAndPublish.yml

```
@@ -56,6 +58,16 @@ jobs:
       run: ci/scripts/setBuildVersionVars.ps1
     - name: Set scons args
       run: ci/scripts/setSconsArgs.ps1
+    - name: set cache key for SCons MSVC Cache
+      shell: bash
+      run: echo "SCONS_CACHE_KEY=${RUNNER_OS}-${ImageVersion}" >> $GITHUB_ENV
+    - id: get-scons-msvc-cache
+      name: Get SCons MSVC Cache
+      uses: actions/cache/restore@v4
```

Why not use the default cache action here?
```suggestion
      uses: actions/cache@v4
```

---

Good point. There's not much benefit in using restore/save since the default action seems to behave exactly the same.